examples/integrations/oci/atp-cdi/src/main/resources/application.yaml

```diff
@@ -21,7 +21,17 @@
 server:
   port: 8080
 
+oracle:
+  ucp:
+    jdbc:
+      PoolDataSource:
+        atp:
+          connectionFactoryClassName: oracle.jdbc.pool.OracleDataSource
+          serviceName: "${atp.db.serviceName}"
```

Comment:
Aha; regarding my comment above, I see it gets tricky here (here's the context I hadn't yet run across).  I see that you're using `atp` as a fairly important configuration signifier.  Nevertheless, it remains the case that your data source currently needs to be "named" `atp` (the `atp` "part" in `oracle.ucp.jdbc.PoolDataSource.atp.something`).  I'll stop short of recommending a change, since it is extraordinarily unlikely that someone will have a data source with that name already, but I think my prior comment is still worth considering.

Reply:
This is example code/resource, we don't rely on "atp" datasource config in our code.